ansible/roles/sidekiq/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+  - name: Install monit
```

Comment:
ok i see -- there is definitely already a monit role that you should include instead